.github/workflows/preview.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: yarn install
 
       - name: 🚀 Publish preview
-        run: expo publish --release-channel=pr-${{ github.event.number }} --non-interactive
+        run: expo publish --profile preview --release-channel=pr-${{ github.event.number }} --non-interactive
```

Comment:
Hopefully this + the channels will mean that the preview builds show off the debug menu